src/contracts/TopChef.sol

```diff
@@ -57,10 +56,9 @@ contract TopChef is Chef {
         _allocations[agIndex].autodistribute = newAutoDistribute;
     }
 
-    function removeAllocationGroup(uint256 agIndex) external onlyOwner {
+    function removeAllocationGroup(uint256 agIndex) external activeRewards onlyOwner {
         // Checks
         if (!(agIndex < _allocations.length)) revert IndexDoesNotMatchAllocation();
```

Comment:
Suggest changing this to ```if (agIndex > _allocations.length) revert IndexDoesNotMatchAllocation();```

Reply:
also I think it should be ` if (agIndex >= _allocations.length) revert IndexDoesNotMatchAllocation();`